yarn-project/p2p/src/client/p2p_client.integration.test.ts

```diff
@@ -0,0 +1,422 @@
+// An integration test for the p2p client to test req resp protocols
```

Comment:
Moved from reqresp.integration.test.ts as it can actually be used to test the entire p2p client